lib/trento/application/projectors/host_projector.ex

```diff
@@ -275,6 +275,16 @@ defmodule Trento.HostProjector do
     end
   end
 
+  def after_update(
+        %HostRemovedFromCluster{host_id: host_id},
+        _,
+        %{host: %Trento.HostReadModel{cluster_id: nil}}
+      ) do
+    TrentoWeb.Endpoint.broadcast("monitoring:hosts", "host_details_updated", %{
+      id: host_id
```

Comment:
Is this the correct payload to broadcast? `cluster_id` will always be `nil` because of the pattern-matching, is it useful to transmit this?